rke/templates/templates.go

```diff
@@ -86,6 +89,9 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.15.0-rancher0 <=1.15.3-rancher1-1": nginxIngressv18,
 			">=1.15.3-rancher2":                     nginxIngressV115,
 		},
+		Nodelocal: {
+			">=1.8.0-rancher0": nodelocalv116,
```

Comment:
@kinarashah Is this the expected behavior, I need to match this to all versions else it errors out while this template will only be introduced for v1.16.

Reply:
@superseb I think we'd have to add some exception in validation check for this template

Reply:
Opened `https://github.com/rancher/kontainer-driver-metadata/pull/76` to relax the check